src/main/java/org/opensearch/flowframework/transport/CreateWorkflowTransportAction.java

```diff
@@ -97,6 +98,8 @@ protected void doExecute(Task task, WorkflowRequest request, ActionListener<Work
 
         if (request.isDryRun()) {
             try {
+                // generating random workflowId only for validation purpose
+                String uniqueID = UUID.randomUUID().toString();
```

Comment:
Where is this UUID used?

There are multiple variants of random UUIDs, is there a particular pattern required here?

Reply:
removed, I previously used this only for the `dryRun` purpose that Josh added for validation, but instead I just switched workflowId to be @Nullable so it still works.